run_robot_test.sh

```diff
@@ -240,20 +249,20 @@ if command -v yq &> /dev/null
                     then
                         oc_host=${api_server}
                     else
-                        oc_host=$(yq  e '.OCP_API_URL' ${TEST_VARIABLES_FILE})
+                        oc_host=$(yq  e '.OCP_API_URL' "${TEST_VARIABLES_FILE}")
                 fi
 
 
                 if [ -z "${SERVICE_ACCOUNT}" ]
                     then
                         echo "Performing oc login using username and password"
-                        oc_user=$(yq  e '.OCP_ADMIN_USER.USERNAME' ${TEST_VARIABLES_FILE})
-                        oc_pass=$(yq  e '.OCP_ADMIN_USER.PASSWORD' ${TEST_VARIABLES_FILE})
+                        oc_user=$(yq  e '.OCP_ADMIN_USER.USERNAME' "${TEST_VARIABLES_FILE}")
+                        oc_pass=$(yq  e '.OCP_ADMIN_USER.PASSWORD' "${TEST_VARIABLES_FILE}")
                         oc login "${oc_host}" --username "${oc_user}" --password "${oc_pass}" --insecure-skip-tls-verify=true
                     else
                         echo "Performing oc login using service account"
-                        sa_token=$(oc serviceaccounts  new-token ${SERVICE_ACCOUNT} -n ${SA_NAMESPACE})
-                        oc login --token=$sa_token --server=${oc_host} --insecure-skip-tls-verify=true
+                        sa_token=$(oc serviceaccounts  new-token "${SERVICE_ACCOUNT}" -n "${SA_NAMESPACE}")
+                        oc login --token="$sa_token" --server="${oc_host}" --insecure-skip-tls-verify=true
```

Comment:
wasn't it working without double quotes?

Reply:
this was a suggestion by the "shellcheck" github action

Reply:
https://www.shellcheck.net/wiki/SC2248